ad/movement/trusts.md

```diff
@@ -291,6 +349,15 @@ The global catalog can be found in many ways, including a simple DNS query (see
 
 In addition to enumerating trusts, retrieving information about the permissions of trusted principals against trusting resources could also allow for lateral movement and privilege escalation. The recon techniques will depend on the permissions to abuse ([DACL](trusts.md#dacl-abuse), [Kerberos delegations](kerberos/delegations/), etc.).
 
+#### Enumerating Bastion Forests
+1. Using the `ADModule`, one can run Get-ADTrust and look for a trust which has `ForestTransitive` set to `True` and `SIDFilteringQuarantined` set to `False` The goal here is to realize whether SID Filtering is disabled. `Get-ADTrust -Filter {(ForestTransitive -eq $True) -and (SIDFilteringQuarantined -eq $False)}`
+2. For situational awareness purposes it is important to enumerate if the current forest is managed by a Bastion Forest. This can be done by looking for: `ForestTransitive` set to `True` and `SIDFilteringForestAware` set to `True`. In this case, TrustAttributes is also a very good indicator, which holds the value of `0x00000400` (`1024` in decimal) for PAM/PIM trust. Overall, it is `1096` for PAM + External Trust + Forest Transitive.
+3. To enumerate the Shadow Security Principals, its members from the current (Bastion) Forest and privileges it holds in the Production Forests, one can use the following command from the ActiveDirectory module: `Get-ADObject -SearchBase ("CN=Shadow Principal Configuration,CN=Services," + (Get-ADRootDSE).configurationNamingContext) -Filter * -Properties * | select Name,member,msDS-ShadowPrincipalSid | fl`
+The following properties are the most interesting ones:
+- `name` - Name of the shadow principal
+- `member` - Members from the bastion forest which are mapped to the shadow principal. In our example, it is the Domain Administrator of defensiveps.local.
+- `msDS-ShadowPrincipalSid` - The SID of the principal (user or group) in the user/prodcution forest whose privileges are assigned to the shadow security principal.
+
```

Comment:
This part could have been shorter. Modified it.
Also, the part mentioning the ForestTransitive and SIDFilteringForestAware was a bit confusing in my opinion